.flake8

```diff
@@ -2,13 +2,20 @@
 max-line-length = 120
 max-complexity = 10
 ignore =
-    E501,  # line too long, it is covered by pylint
-    E722,  # bare except, bad practice, to be removed in the future
-    F401,  # imported but unused, too many violations, to be removed in the future
-    F811,  # redefinition of unused, to be removed in the future
-    C901   # code flow is too complex, too many violations, to be removed in the future
-    W503   # line break before binary operator effect on readability is subjective
-    W504   # line break after binary operator effect on readability is subjective
+    # line too long, it is covered by pylint
```

Comment:
This issue was caused by the updating of flake8 to 6.
![image](https://github.com/Azure/azure-cli-extensions/assets/18628534/91f821d0-5402-49ad-952a-c0bcb02040fa)